rosdep/base.yaml

```diff
@@ -2627,6 +2627,7 @@ libjsoncpp:
   ubuntu:
     artful: [libjsoncpp1]
     bionic: [libjsoncpp1]
+    focal: [libjsoncpp1]
```

Comment:
It looks like all of our currently supported distributions have the package name as `libjsoncpp1`.  In order to automatically get support future Ubuntu distributions, I'll suggest that we change this whole key to:

```
ubuntu: [libjsoncpp1]
```

Reply:
Good idea. Done in 3a6c777.